web/vite.config.ts

```diff
@@ -4,7 +4,7 @@ import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react-swc";
 import monacoEditorPlugin from "vite-plugin-monaco-editor";
 
-const proxyHost = process.env.PROXY_HOST || "localhost:5000";
+const proxyHost = process.env.PROXY_HOST || "192.168.50.106:5002";
```

Comment:
```suggestion
const proxyHost = process.env.PROXY_HOST || "localhost:5000";
```

Reply:
Oops lol, I have a few other changes I need to make too